poetry_version_plugin/plugin.py

```diff
@@ -26,8 +26,7 @@ def activate(self, poetry: Poetry, io: IO) -> None:
             io.write_error_line(message)
             raise RuntimeError(message)
         if version_source == "init":
-            packages = poetry.local_config.get("packages")
-            if packages:
+            if packages := poetry.local_config.get("packages"):
```

Comment:
Thanks! But this would make it only compatible with Python 3.8, it would probably make sense deprecating below 3.8 but I want to first update it and then deprecate.

Anyway, deprecating would be better in another PR independent of the `if` refactors.

Could you revert this part?

Reply:
Will do

Reply:
<img width="725" alt="image" src="https://github.com/tiangolo/poetry-version-plugin/assets/13209783/19ccba1c-2b64-4d5e-9e10-70b3720e12d3">

Now it looks like that Python `version < 3.7 `has to happen earlier?

Reply:
> <img alt="image" width="725" src="https://user-images.githubusercontent.com/13209783/252384964-19ccba1c-2b64-4d5e-9e10-70b3720e12d3.png">
> Now it looks like that Python `version < 3.7 `has to happen earlier?

Due to rebase issue

Reply:
Fixed